ethereum/vm.py

```diff
@@ -514,10 +548,22 @@ def vm_execute(ext, msg, code):
             to = utils.encode_int(to)
             to = ((b'\x00' * (32 - len(to))) + to)[12:]
             extra_gas = (not ext.account_exists(to)) * opcodes.GCALLNEWACCOUNT + \
-                (value > 0) * opcodes.GCALLVALUETRANSFER
+                    (value > 0) * opcodes.GCALLVALUETRANSFER + \
+                    ext.post_anti_dos_hardfork() * opcodes.CALL_SUPPLEMENTAL_GAS
+                    # ^ EIP150 Increase the gas cost of CALL to 700
```

Comment:
I agree with your sentiment, and I actually shot my self in the foot during development because of this. But I don't think changing the style is in scope of this PR.
(btw: the foot is fine again...)